frontends/concrete-python/tests/execution/test_bitwise.py

```diff
@@ -11,13 +12,23 @@
 from concrete.fhe.dtypes import Integer
 from concrete.fhe.values import ValueDescription
 
-cases = []
+operations = [
+    ("&", lambda x, y: x & y),
+    ("|", lambda x, y: x | y),
+    ("^", lambda x, y: x ^ y),
+]
+
+cases: list[
+    Tuple[
+        tuple[str, Callable], int, int, Tuple[int, ...], Tuple[int, ...], None | fhe.BitwiseStrategy
```

Comment:
Does `None | fhe.BitwiseStrategy` syntax is supported in Python 3.8?

Reply:
No it's in 3.10 https://peps.python.org/pep-0604/